packages/core/src/components/cv-combo-box/cv-combo-box.vue

```diff
@@ -206,10 +206,12 @@ export default {
   },
   methods: {
     clearFilter() {
+      this.internalUpdateValue('');
       this.filter = '';
```

Comment:
It should be possible to remove this.filter = ''; as it will be set in internalUpdateValue

Reply:
I won't get updated in `internalUpdateValue()` because in `internalUpdateValue()` it won't update `this.filter` unless it can find it in `this.dataOptions`

```
const filterOption = this.dataOptions.find(item => item.value === val);
if (filterOption) {
   this.filter = filterOption.label;
}
```

Would it be best to keep `this.filter = '';` inside `clearFilter()` and not edit the `internalUpdateValue` function?

Reply:
Your right, thanks for checking.